README.md

```diff
@@ -209,6 +209,7 @@ license is included in the file COPYING.
 ### Citation Information
 
 Andy Grauch, Brendan Billingsley, Chris Chalstrom, Danielle Harper, Ian
-Truslove, Jonathan Kovarik, Luis Lopez, Miao Liu, Michael Brandt, Stuart Reed
-(2013): Arctic Data Explorer SOLR Search software tools. The National Snow and
-Ice Data Center. Software. http://ezid.cdlib.org/id/doi:10.7265/N5JQ0XZM
+Truslove, Jonathan Kovarik, Luis Lopez, Miao Liu, Michael Brandt, Stuart Reed,
+Julia Collins, Scott Lewis
+(2023): Arctic Data Explorer SOLR Search software tools. The National Snow and
+Ice Data Center. Software.
```

Comment:
Removed the DOI because that URL doesn't seem to work.  I did try `https://doi.org/10.7265/N5JQ0XZM`, and it does resolve... to the old BITBUCKET repo for this project.

Should I create a new DOI for this repository and include it here instead?  Or just omit it?

Reply:
DOI status: Scott will write a new story to update the DOI.